Design.md

```diff
@@ -0,0 +1,26 @@
+Overview
+    - This app gives players and teams an environment where they can seamlessly review and breakdown film from games / practices all in one place. It can be used for both internal development and for detailed scouting on future opponents.
+
+Features:
+1. Team Specific Accounts
+    - User notes with this team affiliation are all put in one place.
+    - This means scouting can be done without worrying about another team user seeing it.
+2. User Roles
+    - Users who are verified with a team and can post to that team's account
+    - Users who are not verified with a team and will only be posting on their personal account
+3. Watch Games in App
+    - In a perfect world, the video is playing in the same browser as the app is running.
+    - However, due to cose we might have to use external browser like youtube to run videos.
+        - If this is the case then make it easy to link film clip with timestamp of video
+4. Create Film Clips
+    - While watching a video, user is able to clip part of a video and make a note about what happens
+    - If this user is verified with a team, when another team member is watching the same video they will be able to see the note.
+    - Film clips are kept in a team-wide / user-wide database that can be searched / sorted by keywords (ie. players involved, topic discussed (ie. deep defense), teams, etc...)
+5. Allow Highlights to be Shared
+    - Plays that are deemed highlights while watching can easily be shared
+    - User has the option to include notes along with clip or if they only want to share the clip
+
+Rough Design
+![Home/Game Review](design-draft-images/home:game-review.jpg)
```

Comment:
So, I can't leave comments on the picture so i'll leave them here:

Home Page:
1. I assume "Film App" in the top right is like the name of the app, and will have the logo or whatever?
2. Is the + button that links to "game review" separate from "Your Plays"? If so, i assume "Your Plays" links to a page where we just filter all the clips found for that user's plays?
3. The three activity feeds are interesting, I like the design of them -- you've drawn the design out here for a computer but having them be separate like that means it'd be easy to design the layout in a responsive way (google "responsive web design") where they all collapse to a single column on mobile.

Game Review:
1. This should probably called just like, "Film Review", right? Cuz don't yall watch film of practices and stuff too? Minor nitpick. (P.S. you'll oftentimes see people say "nit" or "nitpick" in pull request reviews, it just means "this is a really minor thing and i don't mean to hurt your feelings")
2. I really like how simple this page is, one big chunk for the video player and a smaller bit for adding a new note. One design decision we should make, though, is whether notes are attached to specific moments in a video or whether they're attached to time intervals.
    - Intuitively, it makes sense for them to be intervals: plays take up time, and mapping "plays" to "start" and "end" times means that you can see a note for the full duration of a play.
    - There might be issues, however, where multiple plays are occurring at the same time. For example, on a "deep defense" play, you might also want to add a shorter "epic sky moment by tyler" at the end. How should we approach that scenario?
    - I think there are pros and cons to both approaches. See what you can think of and reply to this comment (or make a new one, linking to this comment) with what you think we should do.
3. Look at [this](https://developers.google.com/youtube/player_parameters) and [this](https://developers.google.com/youtube/iframe_api_reference) for more about controlling an embedded youtube video. It's a technical thing, so not relevant right now, but it'd be useful to bookmark for later.
4. How will this look on mobile? When watching a clip with someone else's tags, will they appear? Will they pop up and block view of the video? How will it work if they fullscreen the video? Is there a way we can add buttons or elements to say, for example, the iOS video player? How will users add a tag on mobile? Will a full form pop up to overlay the video? Will it ask one question at a time? I really like the desktop design, but i think the mobile design could be an interesting (and difficult!) problem. Probably worth opening a follow-up issue referencing this comment (link to it!) that we'll tackle in a future PR (Pull Request).

Reply:
Home Page:
1. Yep that's where the logo will go I think. It will also serve as a quick link back to the home page when a user is somewhere else on the site.
2. I cut the 'your plays' and the plus to make the nav bar easier to understand.
3. Yep that's what I was envisioning, I always neglect the mobile view even though that is probably the most important one. I am adding a mobile view for each screen on this draft.

Film Review*
1. You're totally right. (amended)
2. I created a completely new section for this answer in the my workup. See #5 in features
3. Bookmarked, those will be very helpful.
4. Have been doing some thinking about this, I will follow up with another pull request soon dedicated to this topic

Reply:
I actually went ahead and added a first thought on how to handle the mobile view for the film review page in #8 in features